tests/publiccloud/run_ltp.pm

```diff
@@ -103,22 +103,66 @@ sub run {
 
     select_host_console();
 
+    ($args->{my_provider}, $args->{my_instance}) = $self->prepare_instance($args);
+
+    my $instance = $args->{my_instance};
+    my $provider = $args->{my_provider};
```

Comment:
As `$args` is not used later, we can simply have:

```suggestion
    my ($provider, $instance) = $self->prepare_instance($args);
```

Reply:
this is not merged yet https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/22802 !